fern/docs/pages/docs/writing-content.mdx

```diff
@@ -1,10 +1,80 @@
-## Configuring content with docs.yml
+## Add content with Markdown or MDX files
 
-Every Fern Docs website has a special configuation file called `docs.yml`.
+Add pages manually to your documentation by creating [Markdown](https://www.markdownguide.org/) (`.md`) or [MDX](https://mdxjs.com/) (`.mdx`) files and linking to them in [docs.yml](#configure-content-with-docs-yml).
 
-<CodeBlock title="docs.yml">
+<CodeBlock title="Example folder structure in docs.yml">
+```bash
+fern/
+├─ fern.config.json
+├─ docs.yml
+└─ pages/
+  ├─ welcome.mdx
+```
+</CodeBlock>
+
+### Custom components
+Fern has a built-in component library you can use in Markdown and MDX files. [Explore the components.](/generate-docs/component-library/)
+
+### Frontmatter
+
+You can optionally use frontmatter for individual page configuration.
+
+Frontmatter must be added to the top of a `.md` or `.mdx` file, before the rest of the content. Use sets of three dashes to indicate the beginning and end of your frontmatter, as shown below.
+
+<CodeBlock title="Example frontmatter in .md or .mdx file">
+```mdx
+---
+title: Fern Docs Quickstart
+description: Learn how to get started building a documentation site for your product with Fern. Automatically generate a API Reference using your OpenAPI specification, configure site navigation, customize styles, and write documentation in Markdown and MDX. Fern's magic puts it all together and builds an attractive site you can host yourself, or host free with Fern.
+editThisPageUrl: https://github.com/fern-api/fern/blob/main/fern/docs/pages/docs/quickstart.mdx
+image: https://github.com/fern-api/fern/blob/main/fern/docs/images/logo-green.png
```

Comment:
It isn't clear to me just looking at this how the image would be used. I think it would be helpful to have a "schema" for frontmatter

```
title:
  type: optional<string>
  docs: use this .....
description: 
   ..... 
```

we plan to make a schema component. until we do, we can match how we do "type" and "default" later in this page.